test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -69,5 +68,17 @@ Describe "Start-Process" -Tags @("CI","SLOW") {
 	$dirEntry.Length | Should BeGreaterThan 0
     }
 
+    It "Should give an error when Verb parameter is used" -Skip:(-not $IsCoreClr) {
+    	$err = $null
+        try 
+	{
+            Start-Process -Verb runas -FilePath $pingCommand -ArgumentList $pingParam
+	}
+	catch
+	{
+            $err = $_
+	}
+	$err.FullyQualifiedErrorId | Should be "NotSupportedException,Microsoft.PowerShell.Commands.StartProcessCommand"
```

Comment:
You are still not following the guideline for validating expected exception. Please see the try/catch example in https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md